MapboxCoreNavigationTests/GeometryTests.swift

```diff
@@ -190,7 +190,7 @@ class GeometryTests: XCTestCase {
         
         // https://github.com/Turfjs/turf/blob/142e137ce0c758e2825a260ab32b24db0aa19439/packages/turf-distance/test.js
         let a = distance(along: line)
-        XCTAssertEqualWithAccuracy(a, 97_159.57803131901, accuracy: 1)
+        XCTAssertEqual(a, 97_159.57803131901, accuracy: 1)
```

Comment:
This won't build using an earlier version.
Try adding: https://github.com/mapbox/turf-swift/blob/e17d1d05c08796536dea0b36a38b9c2bbde0fb06/TurfTests/TurfTests.swift#L7-L12